WordPress/Classes/ViewRelated/Site Creation/Shared/TableViewOffsetCoordinator.swift

```diff
@@ -15,20 +15,37 @@ final class TableViewOffsetCoordinator {
     /// The table view to coordinate
     private weak var tableView: UITableView?
 
+    //// The view containing the toolbar
+    private weak var container: UIView?
```

Comment:
It might be helpful for the name to reflect that it contains the toolbar (or whatever we end up calling it). For example: `footerControlContainer`? 

(Not sure if that's an improvement, because naming is hard. 😅)